Resources/Public/Javascript/3DViewer/main.js

```diff
@@ -0,0 +1,1126 @@
+//Supported file formats: OBJ, DAE, FBX, PLY, IFC, STL, XYZ, JSON, 3DS, glTF
```

Comment:
Please add comment with license information, authors and link to the viewer component's GitHub: https://github.com/slub/3d-viewer

Reply:
Since it is code from @thedworak, he should decide which license to use. In order to be able to integrate in Kitodo, it has to be a GPL-compatible license or the GPL itself. Could you please ask him for license information?